### ISSUE TITLE
Experiment 6: Run EfficientNet on Cifar

### ISSUE THREAD

**Issue by hankyul2** [hankyul2/EfficientNetV2-pytorch#2]
Run EfficientNet on cifar10/100

- Model: EfficientNet-S, M, L, XL
- Dataset: Cifar10

- [x] Prepare cifar setup  
- [x] Prepare ImageNet Pretraining setup 
- [x] Run Model on Cifar10  
- [x] Run Model on Cifar100

**Comment by iamanigeeit** [comment]
I am training EfficientNetV2 without pretrained weights on CIFAR-100 and... thank you for this! Your optimizer and scheduler settings are the only ones that can make the loss converge!

Because i am doing from scratch and not finetuning, my learning rate is 0.1 and epochs=100. It would be good if you can share the finetuning results.

**Comment by hankyul2** [comment]
Hi @iamanigeeit 

Thank you for your interesting question. I already shared the finetuning results such as checkpoint and training logs on [the experiment result section](https://github.com/hankyul2/EfficientNetV2-pytorch#experiment-results). If you click the tf.dev link, you can find the full val top-1/5 accuracy using tensorboard.dev shown in the below graph.
![image](https://github.com/hankyul2/EfficientNetV2-pytorch/assets/31476895/35e7ffad-961f-4bc2-b997-d4bb80d799bd)

Thank you.

Hankyul.
